Description: Fix insecure default PATH
 Throsten Glaser <t.glaser@tarent.de> discovered that the default PATH
 (absent the environment variable) contains the local directory.
Author: Christian Seiler <christian@iwakd.de>
Bug: http://news.gmane.org/find-root.php?message_id=alpine.DEB.2.20.1607181048300.24083%40tglase.lan.tarent.de
Last-Update: 2016-07-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/include/paths.h
+++ b/include/paths.h
@@ -2,7 +2,7 @@
 #define _PATHS_H
 
 #define _PATH_BSHELL "/bin/sh"
-#define _PATH_DEFPATH "/bin:/usr/bin:"
+#define _PATH_DEFPATH "/bin:/usr/bin"
 
 #define _PATH_DEVNULL "/dev/null"
 
